jscomp/others/release.ninja

```diff
@@ -65,7 +65,7 @@ o others/js_vector.cmi : cc others/js_vector.mli | others/belt_internals.cmi oth
 o others/js_weakmap.cmi others/js_weakmap.cmj : cc others/js_weakmap.ml | others/belt_internals.cmi others/js.cmi $bsc
 o others/js_weakset.cmi others/js_weakset.cmj : cc others/js_weakset.ml | others/belt_internals.cmi others/js.cmi $bsc
 o others/jsx.cmi others/jsx.cmj : cc others/jsx.ml | others/belt_internals.cmi others/js.cmi $bsc
-o others/jsxDOM.cmi others/jsxDOM.cmj : cc others/jsxDOM.ml | others/belt_internals.cmi others/js.cmi others/jsx.cmj others/jsxDOMStyle.cmj others/jsxEvent.cmj $bsc
+o others/jsxDOM.cmi others/jsxDOM.cmj : cc others/jsxDOM.res | others/belt_internals.cmi others/js.cmi others/jsxDOMStyle.cmj others/jsxEvent.cmj $bsc
```

Comment:
I believe this seems built properly with res. Could you check it for sure? @cristianoc

Reply:
Looks good to me.